components/src/md3/InfoListItem/InfoListItem.tsx

```diff
@@ -0,0 +1,437 @@
+     *
+     * Default: false
+     */
+    wrapInfo?: boolean;
+
+    /** Whether the subtitle line text should wrap to multiple lines on overflow
+     *
+     * Default: false
+     */
+    wrapSubtitle?: boolean;
+
+    /** Whether the title line text should wrap to multiple lines on overflow
+     *
+     * Default: false
+     */
+    wrapTitle?: boolean;
+
+    /** Style overrides for internal elements. The styles you provide will be combined with the default styles. */
+    styles?: {
+        root?: StyleProp<ViewStyle>;
+        statusStripe?: StyleProp<ViewStyle>;
+        iconWrapper?: StyleProp<ViewStyle>;
+        avatar?: StyleProp<ViewStyle>;
+        icon?: StyleProp<ViewStyle>;
+        mainContent?: StyleProp<ViewStyle>;
+        title?: StyleProp<TextStyle>;
+        subtitle?: StyleProp<TextStyle>;
+        subtitleWrapper?: StyleProp<ViewStyle>;
+        info?: StyleProp<TextStyle>;
+        infoWrapper?: StyleProp<ViewStyle>;
+        divider?: StyleProp<ViewStyle>;
+    };
+
+    /**
+     * Theme value overrides specific to this component.
+     */
+    theme?: $DeepPartial<MD3Theme>;
+};
+
+/**
+ * [InfoListItem](https://brightlayer-ui-components.github.io/react-native/?path=/info/components-documentation--info-list-item) component
+ *
+ * The InfoListItem is a component used to render lists. It extends the basic implementation
+ * of a list item with additional features, such as icons and status stripes and supplies all of
+ * the correct Brightlayer UI styles.
+ */
+export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
+    const {
+        avatar,
+        title,
+        wrapTitle,
+        leftComponent,
+        rightComponent,
+        chevron,
+        divider,
+        subtitle,
+        wrapSubtitle,
+        subtitleSeparator,
+        info,
+        wrapInfo,
+        statusColor,
+        dense, //eslint-disable-line @typescript-eslint/no-unused-vars
+        fontColor, //eslint-disable-line @typescript-eslint/no-unused-vars
+        iconAlign, //eslint-disable-line @typescript-eslint/no-unused-vars
+        iconColor,
+        backgroundColor, //eslint-disable-line @typescript-eslint/no-unused-vars
+        onPress,
+        icon,
+        hidePadding,
+        styles = {},
+        theme: themeOverride,
+        style,
+        ...viewProps
+    } = props;
+    const theme = useTheme(themeOverride);
+    const fontScale = useFontScale();
+    const { disableScaling, maxScale } = useFontScaleSettings();
+    const defaultStyles = infoListItemStyles(props, theme, fontScale);
+
+    const getIconColor = useCallback((): string => {
+        if (iconColor) return iconColor;
+        if (avatar) {
+            return statusColor
+                ? color(statusColor).isDark()
+                    ? Colors.white[50]
+                    : Colors.black[500]
+                : Colors.white[50]; // default avatar is dark gray -> white text
+        }
```

Comment:
Find a way to use theme here

Reply:
I am confused about what colour need to choose from [here](https://www.figma.com/file/UOzL2R0dlgxozVKE4A4yRV/Brightlayer-UI-Tokens-%2B-Components-(in-Material-Design-3)-(Community)?type=design&node-id=12-380&mode=design&t=ztJe6LZREZzATHAL-0) Since Colos.white[50] is `#FFFFFF`. is it Primary50?
<img width="1048" alt="Screenshot 2023-10-12 at 12 42 29 PM" src="https://github.com/etn-ccis/blui-react-native-component-library/assets/133877691/1c9cc3c1-6cc7-4217-b7ab-075f2be41f18">


> Find a way to use the theme here

For colour mapping, I would like to have a separate Jira ticket. Any thoughts?

Reply:
Jira Story BLUI-4795